test_util/cluster_api.py

```diff
@@ -360,7 +362,7 @@ def deploy_test_app_and_check(self, app, test_uuid):
                 msg += "Detailed explanation of the problem: {2}"
                 raise Exception(msg.format(r.status_code, r.reason, r.text))
 
-            assert r.json() == {'username': 'root'}
+            assert r.json() == {'username': self.mesos_user}
```

Comment:
Also note that this "operating_environment" check returns the "container user", which will always be 'root' when run in a docker image whose Dockerfile has USER=root (the default, I believe), even if Marathon's default user is 'nobody', although the sandbox would be owned by 'nobody', and the "host machine user" is still 'nobody'.
We should also test for the "host machine user", which is what `--mesos_user` actually affects.